packages/wrangler/src/dev.tsx

```diff
@@ -1044,6 +1046,7 @@ export async function startApiDev(args: StartDevOptions) {
 				args.accountId ?? configParam.account_id ?? getAccountFromCache()?.id,
 			legacyAssetPaths: legacyAssetPaths,
 			legacyAssetsConfig: configParam.legacy_assets,
+			experimentalAssets: undefined,
```

Comment:
Same here. Tracked elsewhere? Worth a TODO?

Reply:
same reply as in https://github.com/cloudflare/workers-sdk/pull/6370#discussion_r1699567506